.golangci.yml

```diff
@@ -0,0 +1,91 @@
+linters-settings:
+#  govet:
+#    check-shadowing: true
+  golint:
+    min-confidence: 0.8
+  gocyclo:
+    min-complexity: 50
+  staticcheck:
+    go: 1.12
+#  maligned:
+#    suggest-new: true
+#  dupl:
+#    threshold: 150
+#  depguard:
+#    list-type: blacklist
+#    include-go-root: false
+#    packages:
+#      - github.com/sirupsen/logrus
+#      - github.com/bmizerany/assert
+#  funlen:
+#    lines: 70
+#    statements: 40
+#  lll:
+#    line-length: 150
+#  nakedret:
+#    max-func-lines: 10
+
+linters:
+  disable-all: true
+  enable:
+#    - asciicheck
+#    - bodyclose
+#    - deadcode
+#    - depguard
+#    - dogsled
+#    - dupl
+#    - errcheck
+#    - exhaustive
+#    - funlen
+#    - gochecknoglobals
+#    - gochecknoinits
+#    - goconst
+#    - gocritic
+    - gocyclo
+#    - gofmt
+    - goimports
+    - golint
+#    - gomnd
+#    - goprintffuncname
+#    - gosec
+#    - gosimple
+    - govet
+#    - ineffassign
+#    - interfacer
+#    - lll
+#    - maligned
+#    - misspell
+#    - nakedret
+#    - noctx
+#    - nolintlint
+#    - prealloc
+#    - rowserrcheck
+#    - scopelint
```

Comment:
> Scopelint checks for unpinned variables in go programs

A linter for the dangerous bug of using the scope variable of a for loop in a goroutine without passing or pinning it.